content/millennium/r4/care-provision/nutrition-order.md

```diff
@@ -92,7 +92,7 @@ List an individual NutritionOrder by its id:
 
```

Comment:
why not added patient persona examples for search?

Reply:
Search example is already included

Reply:
Can you please point me there. I am not seeing patient search param with Patient person
something similar to

GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/NutritionOrder?patient=12724066

Reply:
https://github.com/cerner/fhir.cerner.com/pull/636/files?file-filters%5B%5D=.md#diff-3132de7f731edec7bd81b6ca7d72b356c5450b48cf211eb6e263816611b2ba30R74
Since i am using the same patient , you are not able to see the diff